### ISSUE TITLE
Crashing the editor on creating an undefined variable reference

### ISSUE THREAD

**Issue by SimeonC** [codesandbox/codesandbox-client#861]
# 🐛 bug report

## How has this issue affected you? What are you trying to accomplish?
I'm just playing with code and the editor will become unresponsive.

In the linked sandbox go to line 82 and change it to `const { isSelectedw } = this.props;`
Wait for a few seconds and the page will crash/hang.

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  |  PROD-1527263059-898b289 |
| Browser | Google Chrome Version 66.0.3359.181 (Official Build) (64-bit) |
| Operating System | Windows 10 Home 64bit version 1709 |

**Comment by arthurdenner** [comment]
Hey, @SimeonC. is this still happening to you? I tried to reproduce it a few times, but I couldn't. After changing line 82, as you described, I only got `isSelected is not defined` (which was expected).

PS: If this is not happening anymore, could you please close the issue?

**Comment by SimeonC** [comment]
Yea, it still happens for me. Though I did notice it only occurs if you have the preview open.
Heres a (rather large) screen cap.
After I took this it said: `index.js:26 Could not consume error: Error: Something is *really* wrong.
    at overlay.js:272` in the browser console.

You're right that `isSelected is not defined` appears in the Codesandbox console, but it seems to keep trying which makes the editor really sluggish - it doesn't crash immediately now with the new update - it does if you leave it for a bit. It might pay to do something to stop it just retrying if an error like that occurs as I got 6000+ of that error in the codesandbox console which does make trying to correct the error rather difficult as it locks up the browser thread.

![image](https://user-images.githubusercontent.com/1085899/41046127-c5ffd284-69e4-11e8-890e-e52c20072a49.png)

**Comment by arthurdenner** [comment]
Thanks for providing more information, @SimeonC. I'll try to find out why this is happening.

**Comment by CompuIves** [comment]
Hey @schaable, to be sure, what version of CodeSandbox are you on? You can find it in the sidebar on the bottom.

**Comment by schaable** [comment]
![screenshot 2018-08-07 10 28 33](https://user-images.githubusercontent.com/1165424/43778663-aba29c32-9a2c-11e8-94ae-c7c966885d4c.png)
It may not be related with the config file from prettier, since I was able to delete the file before the browser crashed (it takes a bit longer to crash on Firefox).

**Comment by lbogdan** [comment]
Hey @SimeonC , is this still happening for you? I tried to replicate, but I only got those "Uncaught ReferenceError: isSelected is not defined" errors in a loop (which I think it's actually a sandbox issue, not a CodeSandbox one), which are now bounded to 900, (probably) because of the loop protection.

If it's not an issue anymore, we'd appreciate you helping us to keep the issue count low by closing this! 🙂
